utility/ThinCompilationsDatabase.py

```diff
@@ -108,8 +108,11 @@ def queryListOfHeaders(command):
     proc=subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     headerlist=[]
     for line in proc.stdout:
-        if matchesHeader(line.strip(), headerlist):
```

Comment:
I think you need to move the `check / decode` before this. See also the issue mentioned by @shahor02 at https://alice-talk.web.cern.ch/t/o2checkcode-o2-systematic-failure/259/1